docs/csharp/misc/cs0077.md

```diff
@@ -39,7 +41,11 @@ class M
       o1 = new C();  
       o2 = new S();  
   
-      s = o2 as S;  
+      // Use pattern matching instead of as
```

Comment:
We should still present the code that generates the error:
```suggestion
      s = o2 as S;    // CS0077, S is not a reference type

      // Use pattern matching instead of as
```

Reply:
We only need to remember (if applying this suggestion) to remove that comment from where it is now.

Reply:
Oh sure. I just updated it. I think it is now completed. I have also validated from console app. it gives error for `as` and then it shows `is` as a fix.